sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/Utils.java

```diff
@@ -115,20 +125,67 @@ public static String createOdataFilterForTags(String tagName, String tagValue) {
      * Download a file asynchronously.
      *
      * @param url the URL pointing to the file
-     * @param retrofit the retrofit client
+     * @param httpPipeline the http pipeline
      * @return an Observable pointing to the content of the file
      */
+    public static Mono<byte[]> downloadFileAsync(String url, HttpPipeline httpPipeline) {
```

Comment:
Why it is in main? I assume this is only needed in test?

Reply:
It's required by [AppServiceCertificateImpl](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/appservice/mgmt/src/main/java/com/azure/resourcemanager/appservice/implementation/AppServiceCertificateImpl.java#L123).